### DISCUSSION TITLE
Chameleon Circuit Issue

### DISCUSSION THREAD

**Discussion by Yamchump36** [eccentricdevotion/TARDIS#840] [Q&A] [answered]
I keep trying to set this as my custom chameleon circuit exterior, but every time I try it just gives me the error I listed below. I even changed the config option to allow all blocks for the chameleon circuit, but it still doesn't work. What am I doing wrong?
![chameleon](https://github.com/eccentricdevotion/TARDIS/assets/158003519/ea33de81-fd77-4e76-ac15-c1b1aee318f1)
![error](https://github.com/eccentricdevotion/TARDIS/assets/158003519/4941ce4d-67c4-4266-a424-297f54e0a263)

**Comment by eccentricdevotion** [comment]
This was fixed fairly recently. Have you updated?

**Reply by eccentricdevotion** [reply]
04db0b76c2df413d9f363408a0fb157aac27f71b

**Reply by Yamchump36** [reply]
Idk, I'd have to check with the server owner to see what version is being used.

**Comment by eccentricdevotion** [comment]
OK, I will do some testing on my end...
What is the black block in your screenshot? A Coal Block?

**Reply by Yamchump36** [reply]
The black block is a redstone lamp

**Comment by eccentricdevotion** [comment, marked as answer]
Fixed in build 2848
